doc/introduction/importing_circuits.rst

```diff
@@ -0,0 +1,272 @@
+
+    Here, the ``measurements`` argument overrides the terminal measurements in the Qiskit circuit.
+    See the :func:`~pennylane.from_qiskit` documentation for more details.
+
+The resulting PennyLane circuit can be executed directly:
+
+>>> pl_circuit_from_qk()
+[tensor(0., requires_grad=True), tensor(1., requires_grad=True)]
+
+It can also be visualized using PennyLane's :func:`~pennylane.draw` utility:
+
+>>> print(qml.draw(pl_circuit_from_qk)())
+0: ──H─╭●─╭||─┤  <Y>
+1: ────╰X─╰||─┤  Var[Z]
+
+OpenQASM
+~~~~~~~~
+
+An equivalent quantum circuit can be expressed in OpenQASM 2.0 as follows:
+
+.. code-block:: python
+
+    oq_circuit = (
+        """
+        OPENQASM 2.0;
+        include "qelib1.inc";
+        qreg q[2];
+        creg c[2];
+
+        h q[0];
+        cx q[0], q[1];
+
+        measure q -> c;
+        """
+    )
+
+We can import this circuit into PennyLane using the PennyLane-Qiskit plugin once more:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    pl_template_from_oq = qml.from_qasm(oq_circuit)
+
+    @qml.qnode(qml.device("default.qubit"))
+    def pl_circuit_from_oq():
+        pl_template_from_oq(wires=[0, 1])
+        return qml.expval(qml.Y(0)), qml.var(qml.Z(1))
+
+The result is as follows:
+
+>>> print(qml.draw(pl_circuit_from_oq)())
+0: ──H─╭●──┤↗├─┤  <Y>
+1: ────╰X──┤↗├─┤  Var[Z]
+
+Quil
+~~~~
+
+PennyLane also offers convenience functions for importing circuits from `pyQuil
+<https://pyquil-docs.rigetti.com/en/stable/index.html>`__ or Quil representations. Both of these
+require the `PennyLane-Rigetti <https://docs.pennylane.ai/projects/rigetti/en/stable/>`__ plugin,
+which can be installed using:
+
+.. code-block::
+
+    pip install pennylane-rigetti
+
+We begin with a familiar pyQuil `Program
+<https://pyquil-docs.rigetti.com/en/stable/apidocs/pyquil.quil.html#pyquil.quil.Program>`__:
+
+.. code-block:: python
+
+    import pyquil
+
+    pq_program = pyquil.Program()
+    pq_program += pyquil.gates.H(0)
+    pq_program += pyquil.gates.CNOT(0, 1)
+
+This ``Program`` can be converted into a PennyLane quantum function using the
+:func:`~pennylane.from_pyquil` function:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    pl_template_from_pq = qml.from_pyquil(pq_program)
+
+    @qml.qnode(qml.device("default.qubit"))
+    def pl_circuit_from_pq():
+        pl_template_from_pq(wires=[0, 1])
+        return qml.expval(qml.Y(0)), qml.var(qml.Z(1))
+
+The resulting PennyLane circuit is:
+
+>>> print(qml.draw(pl_circuit_from_pq)())
+0: ──H─╭●─┤  <Y>
+1: ────╰X─┤  Var[Z]
+
+.. note::
+
+    Quantum circuits expressed in Quil can be imported in a similar way using
+    :func:`~pennylane.from_quil`.
+
+
+Importing quantum operators
+---------------------------
+
+Sometimes, it is preferable to import a single operation from a framework instead of an entire
+quantum circuit. This can save you some keystrokes and serve as a helpful crutch for understanding
+an individual component of a circuit.
```

Comment:
```suggestion
As well as circuits, it can be useful to import operators defined in other frameworks into
PennyLane. This can be useful for workflows that involve calculating the expectation
value of an observable. By mapping to PennyLane, we can make the workflow differentiable
while maintaining access to features like :mod:`grouping <pennylane.pauli>` for
hardware-efficient execution.
```
Just want to position it more towards expectation values.

Reply:
Sounds good! Applied. 🚀